### PR TITLE
EOS-24098: Configure log directory for reset and cleanup

### PR DESCRIPTION
**Pull request by selvakumaar5496**
# Problem Statement
- Configure log directory for reset and cleanup

# Design
- Reset and cleanup configured log dir

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -260,11 +260,12 @@ def config(config_template: str):
...
-        os.makedirs('/var/log/cortx/utils/message_bus', exist_ok=True)
-        os.chmod('/var/log/cortx/utils/message_bus', 0o0777)
-        Path('/var/log/cortx/utils/message_bus/message_bus.log').touch( \
+        mb_log_path = Conf.get(cortx_config_index, 'log_dir', '/var/log')
+        os.makedirs(f'{mb_log_path}/cortx/utils/message_bus', exist_ok=True)
+        os.chmod(f'{mb_log_path}/cortx/utils/message_bus', 0o777)
+        Path(f'{mb_log_path}/cortx/utils/message_bus/message_bus.log').touch( \
             exist_ok=True)
-        os.chmod('/var/log/cortx/utils/message_bus/message_bus.log', 0o0666)
+        os.chmod(f'{mb_log_path}/cortx/utils/message_bus/message_bus.log', 0o666)
```
Comment (selvakumaar5496) [inline py-utils/src/setup/utils.py:268]: changed 0o0666 --> 0o666 as codacy reported it as critical issue.

And recent permission is the correct one too, https://stackoverflow.com/questions/50848968/change-permission-mode-for-new-directories-and-files

File: py-utils/src/support/utils_support_bundle.py [hunk 2]
```diff
@@ -40,11 +40,15 @@ class UtilsSupportBundle:
     _default_path = '/tmp/cortx/support_bundle/'
     _tar_name = 'py-utils'
     _tmp_src = '/tmp/cortx/py-utils/'
+    cortx_config_index = 'cortx_config'
```
Comment (sachinpunadikar) [inline py-utils/src/support/utils_support_bundle.py:43]: All changes in this file are not required, as in K8S env, SB collection is not done by the component

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on the comment

File: py-utils/src/support/utils_support_bundle.py (hunk 2 above)
Comment (selvakumaar5496) [reply py-utils/src/support/utils_support_bundle.py]: Done

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**